packages/flutter_tools/lib/src/ios/cocoapods.dart

```diff
@@ -139,6 +138,28 @@ class CocoaPods {
     }
   }
 
+  //Check if you need to run pod install.
+  //In the cases below, the pod install will not be skipped.
```

Comment:
Avoid double negations. Just say pod install will run if any of the below is true